pkg/paragliderpb/paraglider.proto

```diff
@@ -69,6 +69,10 @@ message ResourceDescriptionString {
     string description = 2;
 }
 
+message ResourceString {
```

Comment:
Ah, okay now that I see this here, I think it is a bit out of place. Maybe define this in `orchestrator.go` and just call it `AttachResourceRequest` or something like that? It might be worthwhile doing the same for `ResourceDescriptionString` in the future too...

Reply:
Sounds good. However, I don't think we should `AttachResourceRequest` for it. I think something more generic like `Resource`/`ResourceID / `ResourceWithID` for easy reusability might be more ideal? Thoughts? And we also have an `AttachResourceRequest` in the protobufs already in use

Reply:
How about we go with `ResourceID`. Sorry for asking for further changes!

Reply:
No worries at all. Will fix

Reply:
Updated